themes/bootstrap3/templates/RecordDriver/DefaultRecord/versions-link.phtml

```diff
@@ -4,7 +4,9 @@
     return;
   }
   if (isset($this->tabs['Versions'])) {
-    $url = !empty($this->full) ? $this->recordLinker()->getTabUrl($this->driver, 'Versions') . '#tabnav' : '#versions';
+    $url = !empty($this->full)
+      ? ($this->recordLinker()->getTabUrl($this->driver, 'Versions', [], ['query' => ['sid' => $this->searchId], 'fragment' => 'tabnav']) . '#tabnav')
```

Comment:
Are we appending the fragment twice here?

Reply:
Actually, no. The parameter didn't work. I misread the code thinking that it would, but forgot to remove the extra parameter.